src/app/utils/utils.js

```diff
@@ -411,3 +411,16 @@ export const formatFriendlyDateToNow = (date) => {
   });
   return formattedDate.concat(" ago");
 };
+
+export const canAdministerModelAccess = (userName, modelStatusData) => {
```

Comment:
As this is a util lets change `modelStatusData` here to just be the `users` value as we don't use anything other than this.

Can you then also add a docbloc of what the values look like for both arguments.